### PR TITLE
HTML/CSS files added, route created

### PR DESCRIPTION
**Pull request by zaiziclebby**
## Changes in this PR

-  Created sign-out.html file and built using GDS components
- Create route for sign-out page 
- Create sign-out.css file and linked to the base.html 
-  Added minimal custom css class to GDS button 

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-442
### After
Screenshot 2023-11-08 at 21 25 41

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/main/routes.py [hunk 1]
```diff
@@ -157,6 +157,11 @@ def start_page():
     return render_template("start-page.html")
 
 
+@bp.route("/sign-out", methods=["GET"])
```
Comment (anthonyhashemi) [inline app/main/routes.py]: "Sign Out" seems misleading as this view doesn't sign a user out, could we change this to "signed out" as well as anywhere else we refer to sign out?

File: package-lock.json [hunk 2]
```diff
@@ -14,6 +14,16 @@
         "stylelint-selector-bem-pattern": "^3.0.1"
       }
     },
+    "node_modules/@aashutoshrathi/word-wrap": {
```
Comment (anthonyhashemi) [inline package-lock.json]: Do we need these changes in the package-lock.json? I dont see any changes to package.json?

Comment (zaiziclebby) [reply package-lock.json]: I could just delete the lock file if that works @anthonyhashemi

**Comment by zaiziclebby** [comment]
Have deleted the SCSS footer file. @colinbowen @anthonyhashemi

**Review by anthonyhashemi — APPROVED** [review]

File: package-lock.json (hunk 2 above)
Comment (anthonyhashemi) [reply package-lock.json]: just need to revert the changes in it, not delete it

File: app/templates/base.html [hunk 3]
```diff
@@ -18,6 +18,7 @@
    <!-- custom css stylesheet -->
    <link rel="stylesheet" type="text/css" src="{{ url_for('static', filename='poc-search.css') }}" />
    <link rel="stylesheet" type="text/css" src="{{ url_for('static', filename='start-page.css') }}" />
+   <link rel="stylesheet" type="text/css" src="{{ url_for('static', filename='sign-out.css') }}" />
```
Comment (anthonyhashemi) [inline app/templates/base.html]: needs updating to signed-out.css

**Pull request merged**